.gitignore

```diff
@@ -9,4 +9,7 @@ react-prototype/npm-debug\.log
 app/config/orderlyweb/go_signal
```

Comment:
Can these app/ lines come out now?

Reply:
The slight weirdness is that if you run `./scripts/run-development-dependencies.sh` from within the `app` dir, these things have to exist in the `app` directory because the `docker-compose` file has a reference to `$PWD`. But now I'm wondering if we can just set a different env var in the script that calls `docker compose`. Let me look into this.

Reply:
Ok this is much nicer :slightly_smiling_face:  https://github.com/vimc/montagu-webapps/pull/432/commits/37f3d187572bf425332a91b6c869dda554b87026